project/RootSettings.scala

```diff
@@ -5,18 +5,10 @@ package akka
 
 import sbt._
 import sbt.Keys._
-import akka.ValidatePullRequest.validatePullRequest
-import sbtunidoc.Plugin.UnidocKeys.unidoc
-import com.typesafe.sbt.site.SphinxSupport
-import com.typesafe.sbt.site.SphinxSupport.Sphinx
 
 object RootSettings extends AutoPlugin {
 
-  lazy val docs = ProjectRef(file("."), "akka-docs")
-
   // settings for root project goes here
   override val projectSettings = Seq(
-    validatePullRequest <<= validatePullRequest.dependsOn(unidoc in Compile),
-    validatePullRequest <<= validatePullRequest.dependsOn(SphinxSupport.generate in Sphinx in docs)
   )
 }
```

Comment:
> and BTW: I think we might need an automated test suite for our build as well ;-)

This can not be verified without running the actual tasks because we started to use dynamic settings and tasks. But we could do a report after a run and validate if the required tasks have been run after the fact.